odoo/tools/template_inheritance.py

```diff
@@ -217,6 +217,26 @@ def extract(spec):
                     if child.get('position') == 'move':
                         child = extract(child)
                     node.addprevious(child)
+            elif pos == 'content':
+                ''' Replace the entire content of an element '''
+                node_children = list(node) # Target's children
+                spec_children = list(spec) # Source's children
+                for child in node_children:
+                    child.getparent().remove(child)
+                node.text = None
+
+                # Add xpath content
+                if spec.text is not None:
+                    node.text = spec.text.lstrip()
+                for child in spec_children:
+                    node.append(child)
+
+                # If the new content has children, right strip the tail of the last one
+                if len(node_children) > 0 and node_children[-1].tail is not None:
+                    node_children[-1].tail = node_children[-1].tail.rstrip()
```

Comment:
Isn't **node** which contains what we want to show on the screen ? So, some lines before, I cleared its old content, then I added the new one. And these lines just clear the unnecessary blank space. Right ?

Reply:
If it works this way it is because converting node to a list above apparently make it still work when the list is modified. But it would be clearer if you do:
```python
children = list(node)
last_child = children[-1]
```
then use `last_child`

Reply:
Yeah, it seems I get the value by reference. But I understand know what you meant. I'll change it :)